### PR TITLE
Simplified add credential help messages.

### LINKED ISSUE #2648: add credentials text is big and not that helpful
During the deployment flow when you select a cloud and go to enter credentials we should leverage existing documentation for that material especially since we know what cloud they're adding for. 

https://jujucharms.com/docs/stable/credentials
https://jujucharms.com/docs/stable/help-aws
https://jujucharms.com/docs/stable/help-azure
https://jujucharms.com/docs/stable/help-google

vs the current material (GCE for example)

    The GCE provider uses OAauth to Authenticate. This requires that you set it up and get the relevant 
    credentials. For more information see  https://cloud.google.com/compute/docs/api/how-tos/ 
    authorization. The key information can be downloaded as a JSON file, or copied from  
    https://console.developers.google.com/project/apiui/credential

### PR DESCRIPTION
**Pull request by kadams54**
Fixes #2648.

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6970/

**Comment by kadams54** [comment]
## QA

You can test this locally using guiproxy.

1. Create a new model.
2. Deploy a charm to the model.
3. Choose any provider in the deployment flow (we will need to check all three).
4. If you have existing credentials, you may need to select "Add new credential..." in the dropdown.
5. Verify that the "Need help?" message above the credentials section links to the generic credentials help page as well as the provider-specific page. (See https://github.com/juju/juju-gui/issues/2648#issue-214702138 for the list of links.)
6. Repeat for each of the three providers.

File: jujugui/static/gui/src/app/views/utils.js [hunk 1]
```diff
@@ -1644,23 +1644,15 @@ YUI.add('juju-view-utils', function(Y) {
...
-              href={'https://console.developers.google.com/apis/credentials'}
-              target="_blank">
-              the Google API credentials page
-            </a>.
-          </p>)
+            Need help? Read more about <a className="deployment-panel__link"
+            href="https://jujucharms.com/docs/stable/credentials"
+            target="_blank" title="Cloud credentials help">credentials in
+            general</a> or <a className="deployment-panel__link"
+            href="https://jujucharms.com/docs/stable/help-google"
```
Comment (frankban) [inline jujugui/static/gui/src/app/views/utils.js]: target="_blank" required here

**Comment by kadams54** [comment]
Thanks @frankban! Targets added. :shipit:

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6975/

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge/2198

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6977/

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**